.prospector/prospector.yaml

```
@@ -0,0 +1,31 @@
+output-format: json
+strictness: high
+test-warnings: false
+doc-warnings: true
+member-warnings: false
+uses:
+- django
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 1

(indentation)